.gitignore

```diff
@@ -55,3 +55,14 @@ ehthumbs_vista.db
 
 # Folder config file
 [Dd]esktop.ini
+.DS_STORE
+examples/*/secrets.yml
+examples/*/terraform.tfstate*
+examples/*/.terraform.lock.hcl
+examples/*/kubeconfig.yml
+examples/*/issue.txt
+examples/*/log.txt
+examples/*/*.png
+examples/*/*.html
+examples/*/issuers.yml
+**/*.swp
```

Comment:
I would not add this on this PR. However, I will leave a few notes:
- there is a section on the top of the file, so the lines related to the `examples` folder should be added there;
- I don't think the `secrets.yml` should be an exception but could be included with placeholders (because in my view these examples are not to be used and any tests should be on a different repository).